flow/scripts/detail_route.tcl

```diff
@@ -63,7 +63,3 @@ write_def $::env(RESULTS_DIR)/5_route.def
 if { [info exists ::env(POST_DETAIL_ROUTE_TCL)] } {
```

Comment:
This does not belong inside the 'if $standalone'.

Reply:
The 'if $standalone' ends on line 22.